src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -128,6 +150,26 @@ public List<MailMessage> fetchEmailMessagesByCreateDate(Date start, Date end, In
   public void sendNewDARCollectionMessage(Integer collectionId)
       throws IOException, TemplateException {
     DarCollection collection = collectionDAO.findDARCollectionByCollectionId(collectionId);
+    List<User> distinctUsers = getDistinctUsers(collection);
+    DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(collection.getDarCode());
+    String researcherName = userDAO.findUserById(dar.getUserId()).getDisplayName();
+    Collection<Dac> dacsInDAR = dacDAO.findDacsForCollectionId(collectionId);
+    List<Dataset> datasetsInDAR = datasetDAO.findDatasetsByIdList(dar.datasetIds);
```

Comment:
We need some safety checking around the bulk calls to ensure we're not passing empty lists.